cumulusci/tasks/github/release.py

```diff
@@ -51,23 +55,22 @@ def _run_task(self):
         except github3.exceptions.NotFoundError:
             pass
         else:
-            message = "Release {} already exists at {}".format(
-                release.name, release.html_url
-            )
+            message = f"Release {release.name} already exists at {release.html_url}"
             self.logger.error(message)
             raise GithubException(message)
 
         # Build tag message
-        message = self.options.get("message", "Release of version {}".format(version))
+        message = self.options.get("message", f"Release of version {version}")
+        message += f"\n\nversion_id: {self.options['version_id']}"
```

Comment:
Now that version_id is optional, you need to be more defensive here:
```suggestion
        if self.options.get("version_id"):
            message += f"\n\nversion_id: {self.options['version_id']}"
```